.ci/code/Versioning_Test/Verify/FromJson.cs

```diff
@@ -47,7 +47,7 @@ public static partial class Verify
         public static TestResult FromJsonDatasets()
         {
             string testFolder = @"C:\ProgramData\BHoM\Datasets\TestSets\Versioning";
-            List<string> versions = new List<string> { "4.2", "4.1", "4.0", "3.3" };
+            List<string> versions = new List<string> { "4.3", "4.2", "4.1", "4.0", "3.3" };
```

Comment:
We might want to keep that list to a minimum based on our analytics of version usage. 
Happy with it staying this way for now though.